crypto/BRCryptoAccount.h

```diff
@@ -37,25 +37,46 @@ extern "C" {
 
     typedef struct BRCryptoAccountRecord *BRCryptoAccount;
 
+    /**
+     * Given a phrase (A BIP-39 PaperKey) dervied the corresponding 'seed'.  This is used
+     * exclusive to sign transactions (BTC ones for sure).
+     *
+     * @param phrase A BIP-32 Paper Key
+     *
+     * @return A UInt512 seed
+     */
     extern UInt512
     cryptoAccountDeriveSeed (const char *phrase);
 
+
+    extern char *
```

Comment:
What about taking in the length of the word list, to confirm that it is sufficiently sized (BIP39_WORDLIST_COUNT)? And/Or what do you think about documenting this function?